ppr-ui/src/composables/fees/FeeSummary.vue

```diff
@@ -352,6 +369,9 @@ export default defineComponent({
     watch(() => props.setRegistrationType, (val: UIRegistrationTypes) => {
       localState.registrationType = val
     })
+    watch(() => props.setTransferType, (val: UITransferTypes) => {
+      localState.transferType = val
+    })
```

Comment:
You won't need this by using the prop directly

Reply:
Changed